node/index.js

```diff
@@ -11,87 +11,151 @@ var plaid = require('plaid');
 // this redirect URI for your client ID through the Plaid developer dashboard
 // at https://dashboard.plaid.com/team/api.
-var PLAID_OAUTH_REDIRECT_URI = envvar.string('PLAID_OAUTH_REDIRECT_URI', '');
-// Set PLAID_OAUTH_NONCE to a unique identifier such as a UUID for each Link
-// session. The nonce will be used to re-open Link upon completion of the OAuth
-// redirect. The nonce must be at least 16 characters long.
-var PLAID_OAUTH_NONCE = envvar.string('PLAID_OAUTH_NONCE', '');
+var PLAID_REDIRECT_URI = envvar.string('PLAID_REDIRECT_URI', '');
 
 // We store the access_token in memory - in production, store it in a secure
 // persistent data store
 var ACCESS_TOKEN = null;
 var PUBLIC_TOKEN = null;
 var ITEM_ID = null;
-// The payment_token is only relevant for the UK Payment Initiation product.
-// We store the payment_token in memory - in production, store it in a secure
+// The payment_id is only relevant for the UK Payment Initiation product.
+// We store the payment_id in memory - in production, store it in a secure
 // persistent data store
-var PAYMENT_TOKEN = null;
 var PAYMENT_ID = null;
 
 // Initialize the Plaid client
 // Find your API keys in the Dashboard (https://dashboard.plaid.com/account/keys)
-var client = new plaid.Client(
-  PLAID_CLIENT_ID,
-  PLAID_SECRET,
-  PLAID_PUBLIC_KEY,
-  plaid.environments[PLAID_ENV],
-  {version: '2019-05-29', clientApp: 'Plaid Quickstart'}
-);
+var client = new plaid.Client({
+  clientID: PLAID_CLIENT_ID,
+  secret: PLAID_SECRET,
+  env: plaid.environments[PLAID_ENV],
+  options: {
+    version: '2019-05-29',
+  }
+});
 
 var app = express();
 app.use(express.static('public'));
-app.set('view engine', 'ejs');
 app.use(bodyParser.urlencoded({
   extended: false
 }));
 app.use(bodyParser.json());
 
 app.get('/', function(request, response, next) {
-  response.render('index.ejs', {
-    PLAID_PUBLIC_KEY: PLAID_PUBLIC_KEY,
-    PLAID_ENV: PLAID_ENV,
-    PLAID_PRODUCTS: PLAID_PRODUCTS,
-    PLAID_COUNTRY_CODES: PLAID_COUNTRY_CODES,
-    PLAID_OAUTH_REDIRECT_URI: PLAID_OAUTH_REDIRECT_URI,
-    PLAID_OAUTH_NONCE: PLAID_OAUTH_NONCE,
-    ITEM_ID: ITEM_ID,
-    ACCESS_TOKEN: ACCESS_TOKEN,
-  });
+  response.sendFile('./views/index.html', { root: __dirname });
 });
 
 // This is an endpoint defined for the OAuth flow to redirect to.
 app.get('/oauth-response.html', function(request, response, next) {
-  response.render('oauth-response.ejs', {
-    PLAID_PUBLIC_KEY: PLAID_PUBLIC_KEY,
-    PLAID_ENV: PLAID_ENV,
-    PLAID_PRODUCTS: PLAID_PRODUCTS,
-    PLAID_COUNTRY_CODES: PLAID_COUNTRY_CODES,
-    PLAID_OAUTH_NONCE: PLAID_OAUTH_NONCE,
-  });
+  response.sendFile('./views/oauth-response.html', { root: __dirname });
+});
+
+app.post('/api/info', function(request, response, next) {
+  response.json({
+    item_id: ITEM_ID,
+    access_token: ACCESS_TOKEN,
+    products: PLAID_PRODUCTS
+  })
+});
+
+app.post('/api/create_link_token', function(request, response, next) {
```

Comment:
nit: Seems like the other routes have links to the docs where relevant. We can add `https://plaid.com/docs/#create-link-token` here